src/components/Text/Annotated/AnnotatedText.tsx

```diff
@@ -2,50 +2,105 @@ import { BroccoliTextGeneric } from "../../../model/Broccoli.ts";
 import { useAnnotationStore } from "../../../stores/annotation.ts";
 import { createSearchRegex } from "../createSearchRegex.tsx";
 import { SegmentedLine } from "./SegmentedLine.tsx";
-import { createLineOffsets } from "./utils/createLineOffsets.ts";
-import { createLineSearchOffsets } from "./utils/createLineSearchOffsets.ts";
-import { getAnnotationsByTypes } from "./utils/getAnnotationsByTypes.ts";
+import { createSearchOffsets } from "./utils/createSearchOffsets.ts";
 import { useDetailUrlParams } from "./utils/useDetailUrlParams.tsx";
 import "./annotated.css";
+import {
+  projectConfigSelector,
+  useProjectStore,
+} from "../../../stores/project.ts";
+import _ from "lodash";
+import {
+  AnnoRepoAnnotation,
+  isLogicalTextAnchorTarget,
+} from "../../../model/AnnoRepoAnnotation.ts";
+import {
+  createFootnoteMarkLineOffsets,
+  createNestedLineOffsets,
+  createPageMarkLineOffsets,
+} from "./utils/createLineOffsets.ts";
+import { LineOffsets } from "./AnnotationModel.ts";
 
 type TextHighlightingProps = {
   text: BroccoliTextGeneric;
   showDetail: boolean;
 };
 
+export const DUMMY_ID = "urn:suriano:file:1697716";
```

Comment:
Kan weg.

Reply:
Fixed in [8988b27](https://github.com/knaw-huc/textannoviz/pull/154/commits/8988b2734d0fa14a4d9f8206dc51dc3b236b484f)